crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -2436,6 +2437,7 @@ pub(crate) mod tests {
 
         let mut request =
             machine.keys_for_upload(&account).await.expect("Can't prepare initial key upload");
+        drop(account); // Release the account lock.
```

Comment:
Let's introduce a subscope like in the rest of the method.

Reply:
(This is done in another commit.)